sycl/doc/extensions/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc

```diff
@@ -96,10 +96,9 @@ enum class register_map_interface_options_enum {
   do_not_wait_for_done_write
 };
 
-struct initiation_interval_key {
-  template <int initiation_interval_value>
-  using value_t = property_value<initiation_interval_key,
-    std::integral_constant<int, initiation_interval_value>>;
+template <int N>
+struct initiation_interval {
+  int initiation_interval_value = N;
```

Comment:
Is this expected to mutate? Otherwise it would be better to have it be `static constexpr`. Also, the naming of the value member seems a little verbose, given it simply repeats the type name. Would `value` be too simple a name?

Reply:
Value is not expected to mutate so I will make it a `static constexpr` and yes I can rename the field to `value` as I don't think that the more verbose name is needed.